spec/ParseUser.spec.js

```diff
@@ -13,6 +13,91 @@ const passwordCrypto = require('../lib/password');
 const Config = require('../lib/Config');
 const cryptoUtils = require('../lib/cryptoUtils');
 
+describe('Parse.User auth after save testing (original object not being retrieved) ', () => {
```

Comment:
Could you remove this new `describe` and move the `it` under the existing  `Parse.User testing'`?